quinn-udp/tests/tests.rs

```diff
@@ -91,6 +91,44 @@ fn ecn_v4() {
     }
 }
 
+#[test]
+#[cfg_attr(target_os = "windows", ignore)]
```

Comment:
I am failing to send an IPv4-mapped IPv6 packet to an IPv4 socket on windows thus far.

Reply:
That definitely works fine in general; we have E2E test coverage for dual-stack servers talking to IPv4 clients. Looks like maybe Windows just doesn't like IP_TOS on IPv6 sockets? Maybe it works with `IPV6_TCLASS`? Let me know if you don't have a Windows machine you can easily test on and I can tinker with it myself.

Error for reference:

> called `Result::unwrap()` on an `Err` value: Os { code: 10022, kind: InvalidInput, message: "An invalid argument was supplied." }

Reply:
> Looks like maybe Windows just doesn't like IP_TOS on IPv6 sockets? Maybe it works with IPV6_TCLASS?

This is the hint I needed. I only applied the actual patch to `unix.rs`. Opposite to *nix, Windows actually errors, when using `IPV6_TCLASS` with an IPv4-mapped IPv6 address. Fixed with https://github.com/quinn-rs/quinn/pull/1765/commits/0b97f4638d059fb242768375284bed82f129528d. Thank you @Ralith!

Reply:
I actually had that backwards; glad it was a useful prompt anyway!